src/qibo/backends/clifford.py

```diff
@@ -28,12 +27,26 @@ class CliffordBackend(NumpyBackend):
     def __init__(self, engine=None):
         super().__init__()
 
-        if engine is None:
-            from qibo.backends import _check_backend
+        if engine == "stim":
+            try:
+                import stim  # pylint: disable=C0415
 
-            engine = _get_engine_name(_check_backend(engine))
+                self._stim = stim
+            except ModuleNotFoundError:  # pragma: no cover
+                raise_error(
+                    ModuleNotFoundError,
+                    "`engine` was set as `stim`, but module was not found. Please install `stim`.",
+                )
```

Comment:
Why not to leave the original error? It's not the exact same, but it's very similar to this one.